app/ui/view_internal.h

```diff
@@ -66,6 +66,15 @@ static const char* review_addrvalue = REVIEW_SCREEN_ADDR_VALUE;
 static const char* review_keyconfig = "Review";
 static const char* review_configvalue = "configuration";
 
+// Review msg string can be customizable in each app
+#if !defined(REVIEW_MSG_TITLE) && !defined(REVIEW_MSG_VALUE)
+#define REVIEW_MSG_TITLE "Please"
+#define REVIEW_MSG_VALUE "review"
+#endif
```

Comment:
Maybe we could split this to be able to define only one of the two. Normally you'd define both, but it avoids undefined behaviours.